### PR TITLE
Session 3 coding assignment

### PR DESCRIPTION
**Pull request by iswarya15**

### CODE REVIEW

File: session 03/src/App.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  "div",
+  { className: "title" },
+  [
+    React.createElement("h1", {}, "Header using React.CreateElement"),
+    React.createElement("h2", {}, "React BootCamp"),
+    React.createElement("h3", {}, "In 3 months"),
+  ]
+);
+
+var course = "React BootCamp";
```
Comment (nitishnivedan) [inline session 03/src/App.js]: should use const here if value is not getting changed. Avoid using var, instead can use let where required.

File: session 03/src/App.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
```
Comment (nitishnivedan) [inline session 03/src/App.js:2]: have a new line between library imports and local file imports for better readability of code

File: session 03/src/App.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    </div>
+  );
+};
+
+// {TitleComponent} => Functions are not valid as a React child. This may happen if you return a Component instead of <Component /> from render. Or maybe you meant to call this function rather than return it.
+
+// {<TitleComponent>} & {<TitleComponent></TitleComponent>} renders the component
+
+console.log("Header using JSX => ", headerUsingJSX);
+console.log("Header using FuncComponent => ", HeaderUsingFuncComponent());
```
Comment (shrey31893) [inline session 03/src/App.js:44]: for your purpose - you can add comment of this output of logging.

**Review by shrey31893 — APPROVED** [review]

File: session 03/src/App.js (hunk 2 above)
Comment (iswarya15) [reply session 03/src/App.js:2]: Added new line

File: session 03/src/App.js (hunk 3 above)
Comment (iswarya15) [reply session 03/src/App.js:44]: sure.

File: session 03/src/App.js (hunk 1 above)
Comment (iswarya15) [reply session 03/src/App.js]: updated.

**Pull request merged**